### DISCUSSION TITLE
Organic Maps routing is BEST

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [organicmaps/organicmaps#9669] [Ideas]
Here https://github.com/orgs/organicmaps/discussions/9659#discussion-7484359 I promised that I would write about how Organic routes. After several relationships, I had to open a new topic.

Organic Maps routes the **BEST** of all the apps that use OSM maps. It routes identically to iGO with OSM maps.
Organic maps routes better than Sygic, HERE,... etc

That's what I wanted to know. GOOD JOB.
'go straight'

**Comment by Petak73** [comment]
No, it's not, because it doesn't take traffic into account.

**Reply by ZeljkoBG** [reply]
I know applications that have Live traffic and route incorrectly both with and without the service.
OM routing without traffic is great, for now.

**Reply by Petak73** [reply]
Maybe but there is a huge gap between "best" and "there are worse" :-))
